addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -91,7 +93,7 @@ const SharedPopupWidget = publicWidget.Widget.extend({
 publicWidget.registry.SharedPopup = SharedPopupWidget;
 
 const PopupWidget = publicWidget.Widget.extend({
-    selector: '.s_popup',
+    selector: ".s_popup:not(#website_cookies_bar)",
```

Comment:
As I refactored the commit to use bootstrap show/hide events, I had to extend the `PopupWidget`. I'm doing this here to avoid calling twice each method on the widget.